components/profile/rating/src/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+class ProfileRating extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      ratingFilledWidth: 0
+    }
+  }
+
+  calculateWidthByRating () {
+    const { maxValue, iconSize, rating, spacingBetween } = this.props
+
+    return (maxValue * (iconSize + spacingBetween) / maxValue) * rating
+  }
+
+  getIcon (Icon, index) {
+    return <div key={index} className='sui-ProfileRating-iconContainer' style={{ minWidth: this.props.iconSize, marginLeft: this.props.spacingBetween }}>
```

Comment:
you might deconstruct iconSize and spacingBeteen here too:
const { iconSize, spacingBetween } = this.props